cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithLocallyBuildCordappAndConfig.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
     ext {
         corda_group = 'net.corda'
-        corda_release_version = '3.0-SNAPSHOT' // TODO: Set to 3.0.0 when Corda 3 is released
+        corda_release_version = '5.0-SNAPSHOT' // TODO: Set to 5.0 when Corda 5 is released
```

Comment:
Use `4.0` instead of `5.0-SNAPSHOT`, because only your "network parameter overrides" should need `5.0`.

Reply:
Fixed in c526c85c1def975567a5e5908ae355d8b02473d5